ci-operator/config/openshift/cluster-nfd-operator/openshift-cluster-nfd-operator-release-4.2.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+canonical_go_repository: github.com/openshift/cluster-nfd-operator
+images:
+- from: base
+  to: cluster-nfd-operator
+promotion:
+  name: "4.2"
+  namespace: ocp
```

Comment:
This should be `disabled: true`, because right now the master branch is promoting into `ocp/4.2`.

Reply:
Where do I have to disable it?

Reply:
Simply add `disabled: true` to the promotion stanza (and rerun prowgen, it will make the jobs slightly different):

```
promotion:
  name:
  namespace:
  disabled: true
```

Reply:
Done, and when master is promoting to 4.3 I can enable the 4.2 config right?

Reply:
Yup, but the switch will happen centrally by tools for all repos.

Reply:
Thanks, good to know.